src/app/content/Content.tsx

```diff
@@ -1,38 +1,21 @@
 import { Box, Skeleton } from '@chakra-ui/react';
 
+import { SelectedCourse } from '../../App';
```

Comment:
this is being imported a few times, should we make a types.ts?

Reply:
I'm not sure where as I don't think a `types` file in the root of the application is fitting. Likely we might move stuff out of `App.tsx` into move specific components which might be a good time to make those `types.ts` as needed.